sdk/nodejs/__tests__/client.spec.ts

```diff
@@ -59,12 +60,16 @@ describe('variable', () => {
     })
 
     it('returns a valid variable object for a variable that is not in the config', () => {
+        // @ts-ignore
+        getBucketingLib().variableForUser.mockReturnValueOnce(null)
```

Comment:
doesnt matter that much but you can also use the `jest.Mocked` type or whatever its called to give you correct typing here